includedBuild/gradleConfiguration/src/main/kotlin/com/badoo/reaktive/configuration/Target.kt

```diff
@@ -13,6 +13,7 @@ enum class Target(
     ALL_LINUX_HOSTED(linuxHosted = true, macOsHosted = false),
     JVM(linuxHosted = true, macOsHosted = false),
     JS(linuxHosted = true, macOsHosted = false),
+    WASM_JS(linuxHosted = true, macOsHosted = false),
```

Comment:
Not sure, but we could reuse the `JS` target for `wasmJs`, and configure it in `JsPlugin`. See [here](https://github.com/badoo/Reaktive/compare/master...arkivanov:Reaktive:wasm#diff-d38bf8ed74b4d0d9b4c0c623a3f79d84d47633e345d8b418ec18aadf311d1702). Up to @CherryPerry.

Reply:
Would prefer separate `WASM_JS` to follow the existing structure. It is a separate target, not part of JS like `node()` and `browser()`.

Both reusing `JsPlugin` and a separate plugin to setup WASM is fine.

Reply:
Well, `wasmJs` also has `browser()` and `node()`. But I'm fine with either way